tests/integration-tests-with-streaming/dockerFullFlow-test.js

```diff
@@ -0,0 +1,265 @@
+                        await configRequestCreator.updateConfig({
+                            streaming_excluded_attributes: 'intermediates,artillery_test'
+                        });
+                    });
+
+                    it('Create the job', async () => {
+                        const validBody = {
+                            test_id: testId,
+                            type: 'load_test',
+                            arrival_rate: 2,
+                            duration: 1,
+                            run_immediately: true,
+                            notes: 'streaming notes'
+                        };
+
+                        createJobResponse = await schedulerRequestCreator.createJob(validBody, {
+                            'Content-Type': 'application/json'
+                        });
+
+                        jobId = createJobResponse.body.id;
+                        should(createJobResponse.status).eql(201);
+                    });
+
+                    it('Consume published event job-created', async () => {
+                        const jobCreatedMessage = await kafkaHelper.getLastMsgs(1);
+                        const valuePublished = JSON.parse(jobCreatedMessage[0].value);
+
+                        should(valuePublished.event).eql('job-created');
+                    });
+
+                    it('Predator-runner posts "done" stats', async () => {
+                        reportId = createJobResponse.body.report_id;
+                        await reportsRequestCreator.subscribeRunnerToReport(testId, reportId, runnerId);
+                        const statsFromRunnerIntermediate = statsGenerator.generateStats(constants.SUBSCRIBER_INTERMEDIATE_STAGE, runnerId, statsTime);
+                        await reportsRequestCreator.postStats(testId, reportId, statsFromRunnerIntermediate);
+
+                        const statsFromRunnerDone = statsGenerator.generateStats(constants.SUBSCRIBER_DONE_STAGE, runnerId, statsTime);
+                        await reportsRequestCreator.postStats(testId, reportId, statsFromRunnerDone);
+                    });
+
+                    it('Consume published event job-finished', async () => {
+                        const jobFinishedMessage = await kafkaHelper.getLastMsgs(1);
+                        const valuePublished = JSON.parse(jobFinishedMessage[0].value);
+
+                        should(valuePublished.event).eql('job-finished');
+                        should(valuePublished.resource.intermediates).be.undefined();
+                        should(valuePublished.resource.artillery_test).be.undefined();
+                        should(valuePublished.resource.aggregate).be.not.undefined();
+                    });
+
+                    it(('Remove excluded streaming attributes from config'), async () => {
+                        await configRequestCreator.updateConfig({
+                            streaming_excluded_attributes: ''
```

Comment:
you can delete this config param no? how do we unset it?

Reply:
true i should call DELETE and not update with empty string

Reply:
done